tools/isobuild/package-source.js

```diff
@@ -422,18 +423,25 @@ _.extend(PackageSource.prototype, {
     utils.ensureOnlyExactVersions(options.cordovaDependencies);
     self.cordovaDependencies = options.cordovaDependencies;
 
-    var sources = _.map(options.sources, function (source) {
-      if (typeof source === "string")
-        return {relPath: source};
+    const sources = options.sources.map((source) => {
+      if (typeof source === "string") {
+        return {
+          relPath: source
+        };
+      }
+
       return source;
     });
 
-    var sourceArch = new SourceArch(self, {
+    const sourceArch = new SourceArch(self, {
       kind: options.kind,
       arch: "os",
       uses: _.map(options.use, splitConstraint),
-      getSourcesFunc: function () { return sources; },
-      nodeModulesPath: nodeModulesPath
```

Comment:
I discovered that it's not used anywhere at all, and is just two extra lines that do literally nothing (as far as I can tell!)

So I decided to change it while I was there. Should I revert this part?